packages/app/src/update/listeners/get-all-development-releases.listener.ts

```diff
@@ -81,9 +37,9 @@ const releasesRequest = async (
   }
 }
 
-const registerGetAllReleasesListener = () => {
+export const registerGetAllDevelopmentReleasesListener = () => {
```

Comment:
I'm not sure that all releases are as Development 😛 I think that we could avoid this prefix 🙃

Reply:
I guess it would be better to keep these prefix

Reply:
I'm not sure 😛 this listener will emit all releases not just filtered (by development)? 🤔

Reply:
https://github.com/mudita/mudita-center/pull/629/commits/7d4164dffb783556c4508f2bc2006b0aef4d25ef